vips/vip-304/bscmainnet.ts

```diff
@@ -0,0 +1,120 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { NETWORK_ADDRESSES } from "../../src/networkAddresses";
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const { bscmainnet } = NETWORK_ADDRESSES;
+
+export const DEV_WALLET = "0x48e9d2128321cbf75cd108321459865357c00f15";
+export const WBNB = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const USDT = "0x55d398326f99059fF775485246999027B3197955";
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+export const BTC = "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c";
+export const vUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const TOKEN_REDEEMER = "0x2367bff367Bc4a83A65EEEe3b9E4D834Ff4f637e";
+
+export const WBNB_AMOUNT_DEV_FUND = parseUnits("1472.197705207413945", 18);
+export const ETH_AMOUNT_DEV_FUND = parseUnits("62.069755574301455", 18);
+export const BTC_AMOUNT_DEV_FUND = parseUnits("1.702705080868696", 18);
+export const USDT_AMOUNT_DEV_FUND = parseUnits("21100", 18);
+
+export const REQUIRED_VUSDC_AMOUNT_FOR_REMAINING_USDC = parseUnits("14046500", 8);
+export const USDC_AMOUNT_ON_TREASURY = parseUnits("16777", 18);
+export const REMAINING_USDC_AMOUNT = parseUnits("334623", 18);
+
+export const BNB_TREASURY = "0xf322942f644a996a617bd29c16bd7d231d9f35e9";
+export const COMMUNITY_WALLET = "0xc444949e0054A23c44Fc45789738bdF64aed2391";
+
+export const ETH_AMOUNT_COMMUNITY_WALLET = parseUnits("1.188225443", 18);
+export const USDT_AMOUNT_COMMUNITY_WALLET = parseUnits("10518.73", 18);
+export const BNB_AMOUNT_COMMUNITY_WALLET = parseUnits("3", 18);
```

Comment:
These funds shouldn't be transferred to the Community wallet, but to the DEV_WALLET

Reply:
[c4535f2](https://github.com/VenusProtocol/vips/pull/288/commits/c4535f23411cec1c7434eb6be958d3e6c96c72b1)